src/Context/Drupal/CoreCkeditor.php

```diff
@@ -40,9 +40,23 @@ protected function ckeditorSetData(string $fieldId, string $newValue)
         $fieldIdSafe = addslashes($fieldId);
         $newValueSafe = addslashes($newValue);
 
+        $editor = "div.js-form-item-$fieldIdSafe-0-value .ck-editor__editable";
+
         $this
             ->getSession()
-            ->executeScript("CKEDITOR.instances['{$fieldIdSafe}'].setData('{$newValueSafe}');");
+            ->executeScript("
+      var domEditableElement = document.querySelector(\"$editor\");
+      if (domEditableElement.CKEditorInstance) {
```

Comment:
Currently, this is causing the issue: `TypeError: Cannot read properties of null (reading 'CKEditorInstance')
          at <anonymous>:3:30 (Behat\Mink\Exception\DriverException)`.

Reply:
Fixed in commit [6b2c35b](https://github.com/Cheppers/drupal-behat-contexts/pull/34/commits/6b2c35b47735f84e63ca43a4a92aa01e8708456e).